README.md

```diff
@@ -68,6 +68,10 @@ To make the steam overlay working, call the `electronEnableSteamOverlay` on the
 require('steamworks.js').electronEnableSteamOverlay()
 ```
 
+[Some users have reported an issue when launching Linux apps](https://github.com/ceifa/steamworks.js/issues/42). A workaround is available, to pass a `--no-sandbox` as an argument to the app launcher in Steamworks --> Steamworks Settings --> Installation --> General Installation --> Launch Options. (Note that `--no-sandbox` makes it unsafe to run third-party code, but as long as you're only running game code under your own control, it should be fine.) A screenshot example of this appears below.
```

Comment:
This only happens on steam deck or linux in general?

Reply:
Back in Sep 2022, I reproduced the issue on Ubuntu 22 and Manjaro 22.

Reply:
I also can confirm that this is an issue that happens to Linux 'in general'.

According to the bug reports that our game ([Friendly Fire](https://store.steampowered.com/app/1311660/Friendly_Fire/)) received from Linux users, it seems to me that `--no-sandbox` currently is the only viable option to push a reliable working Linux version of our Electron-based game.